### DISCUSSION TITLE
First Plugins to Create

### DISCUSSION THREAD

**Discussion by AJStacy** [adzejs/adze#170] [General]
Hey everyone,
What are some plugins for Adze that would be helpful to create first? I am currently working on publishing a file rotator transport plugin, but I'm sure there are others that are important to you. Here are some ideas:

- Loki transport middleware
- OpenTelemetry transport middleware
- Cloudwatch Logs transport middleware
- Google Cloud Logging transport middleware

Any others that might be good to tackle early on?

**Comment by therockstorm** [comment]
Datadog!

**Reply by AJStacy** [reply]
Well, since you’re the first to reply I suppose DataDog needs to be on the list!

**Comment by KingOfTac** [comment]
SignalR plugin would be nice, but I think something that can capture logs from any third-party library would be more beneficial than per-library plugins.

**Reply by AJStacy** [reply]
Adze by default is capable of capturing logs for any third-party library. You just have to write a small amount of middleware code to do so.

**Comment by jpdewing** [comment]
Hi, very nice library, thanks! 

A transport for Loki would be very useful.

**Reply by AJStacy** [reply]
Thanks for the input! It's on my TODO list.
